### PR TITLE
yngshan/fix_missing_tick_wording

### PR DESCRIPTION
**Pull request by yngshan**
- add wording for missing tick issue

### CODE REVIEW

File: src/templates/toolkit/bet/explanation/details/digits.html.tt [hunk 1]
```diff
@@ -4,5 +4,5 @@
 </p>
 
 <p class="hint">
-  <strong>[% l("Note") %]: </strong>[% l("Contracts will be refunded at the purchase price if the following conditions occur: (1) there are less than 2 ticks between the start and end times, or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes.") %]
+  <strong>[% l("Note") %]: </strong>[% l("Contracts will be refunded at the purchase price if the following conditions occur: (1) there are less than 2 ticks between the start and end times, or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes, or (3) for contracts starting in the future, if there are no ticks in the 5 minutes before the contract's start time.") %]
```
Comment (jy-binary) [inline src/templates/toolkit/bet/explanation/details/digits.html.tt]: digits can't start in the future? Doesn't seem relevant to add this text here.

File: src/templates/toolkit/bet/explanation/details/asian.html.tt [hunk 2]
```diff
@@ -7,5 +7,5 @@
     [% l("The average is the average of the ticks, including the first and the last tick.") %]
 </p>
 <p class="hint">
-  <strong>[% l("Note") %]: </strong>[% l("Contracts will be refunded at the purchase price if the following conditions occur: (1) there are less than 2 ticks between the start and end times, or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes.") %]
+  <strong>[% l("Note") %]: </strong>[% l("Contracts will be refunded at the purchase price if the following conditions occur: (1) there are less than 2 ticks between the start and end times, or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes, or (3) for contracts starting in the future, if there are no ticks in the 5 minutes before the contract's start time.") %]
```
Comment (jy-binary) [inline src/templates/toolkit/bet/explanation/details/asian.html.tt]: can Asian bets be forward-starting?

File: src/templates/toolkit/bet/explanation/details/evenodd.html.tt [hunk 3]
```diff
@@ -3,5 +3,5 @@
     [% l("The first tick is the first tick after the contract is processed by our servers.") %]
```
Comment (jy-binary) [inline src/templates/toolkit/bet/explanation/details/evenodd.html.tt]: again, doesn't seem relevant here

File: src/templates/toolkit/bet/explanation/details/spreads.html.tt [hunk 4]
```diff
@@ -6,5 +6,5 @@
     [% l("The amount of profit/loss per point of rise/fall is multiplied by the amount per point. A large amount per point will result in large profits/losses per point of movement of the index.") %]
 </p>
 <p class="hint">
-  <strong>[% l("Note") %]: </strong>[% l("Contracts will be refunded at the purchase price if the following conditions occur: (1) there are less than 2 ticks between the start and end times, or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes.") %]
+  <strong>[% l("Note") %]: </strong>[% l("Contracts will be refunded at the purchase price if the following conditions occur: (1) there are less than 2 ticks between the start and end times, or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes, or (3) for contracts starting in the future, if there are no ticks in the 5 minutes before the contract's start time.") %]
```
Comment (jy-binary) [inline src/templates/toolkit/bet/explanation/details/spreads.html.tt]: spreads can't start in the future, also spreads have no end time

File: src/templates/toolkit/bet/explanation/details/asian.html.tt (hunk 2 above)
Comment (yngshan) [inline src/templates/toolkit/bet/explanation/details/asian.html.tt]: Opps, my bad. I did not aware it need to be contract specific. 

So we should not mentioned about or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes on those path dependent and spread section as well

Comment (jy-binary) [inline src/templates/toolkit/bet/explanation/details/asian.html.tt]: Yes, please review each one and edit accordingly to  each contract's specifics.






On Thu, May 5, 2016 at 6:15 PM -0700, "yngshan" <notifications@github.com> wrote:












In src/templates/toolkit/bet/explanation/details/asian.html.tt:
> @@ -7,5 +7,5 @@
>      [% l("The average is the average of the ticks, including the first and the last tick.") %]
>  </p>
>  <p class="hint">
> -  <strong>[% l("Note") %]: </strong>[% l("Contracts will be refunded at the purchase price if the following conditions occur: (1) there are less than 2 ticks between the start and end times, or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes.") %]
> +  <strong>[% l("Note") %]: </strong>[% l("Contracts will be refunded at the purchase price if the following conditions occur: (1) there are less than 2 ticks between the start and end times, or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes, or (3) for contracts starting in the future, if there are no ticks in the 5 minutes before the contract's start time.") %]



Opps, my bad. I did not aware it need to be contract specific. 



So we should not mentioned about or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes on those path dependent and spread section as well

File: src/templates/toolkit/bet/explanation/details/asian.html.tt [hunk 5]
```diff
@@ -7,5 +7,5 @@
     [% l("The average is the average of the ticks, including the first and the last tick.") %]
 </p>
 <p class="hint">
-  <strong>[% l("Note") %]: </strong>[% l("Contracts will be refunded at the purchase price if the following conditions occur: (1) there are less than 2 ticks between the start and end times, or (2) for contracts of tick duration, if the contract doesn't end within 5 minutes.") %]
+  <strong>[% l("Note") %]: </strong>[% l("Asians contracts will be refunded at the purchase price if the contract doesn't end within 5 minutes.") %]
```
Comment (negar-binary) [inline src/templates/toolkit/bet/explanation/details/asian.html.tt]: shouldn't it be 'Asian contracts' instead?

Comment (yngshan) [inline src/templates/toolkit/bet/explanation/details/asian.html.tt]: I use the same naming as on website :)

**Pull request merged**